features/poller/client-rpc/src/test/java/org/opennms/netmgt/poller/client/rpc/PollerResponseDTOTest.java

```diff
@@ -45,12 +49,16 @@ public PollerResponseDTOTest(PollerResponseDTO sampleObject, String sampleXml) {
 
     @Parameters
     public static Collection<Object[]> data() throws Exception {
+        String localEpoch = ((new Date(0)).toInstant())
```

Comment:
We do similar date manipulation in other XML tests too:
https://github.com/OpenNMS/opennms/blob/9dfa101190a0483f9edc0c83b4f4efceed99399e/opennms-alarms/api/src/test/java/org/opennms/netmgt/alarmd/api/NorthboundAlarmTest.java#L76

Can we make this method generic and move into the `org.opennms.core.test-api.xml` project instead?

Bonus points for refactoring the NorthboundAlarmTest to use it too :)